purchase_cancel_remaining_delivery/views/purchase_order.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="purchase_order_form" model="ir.ui.view">
+        <field name="name">purchase.order.form</field>
+        <field name="model">purchase.order</field>
+        <field name="inherit_id" ref="purchase.purchase_order_form" />
+        <field name="arch" type="xml">
+            <button name="button_done" position="after">
+                <button
+                    name="button_cancel_remaining_delivery"
+                    string="Cancel Remaining Delivery"
+                    type="object"
+                    groups="account.group_account_invoice"
```

Comment:
Why is this button not shown for all purchase users, or the manager, why is it only shown for the invoicing group?

Reply:
Mistake :face_in_clouds: I removed it.
Thanks for your review